Project.swift

```diff
@@ -106,12 +106,52 @@ let project = Project(name: "Mail",
                                   .target(name: "Mail")
                               ]
                           ),
+                          Target(
+                              name: "MailShareExtension",
+                              platform: .iOS,
+                              product: .appExtension,
+                              bundleId: "com.infomaniak.mail.ShareExtension",
+                              deploymentTarget: Constants.deploymentTarget,
+                              infoPlist: .file(path: "MailShareExtension/Info.plist"),
+                              sources: ["MailShareExtension/**",
+                                        "Mail/Views/**",
+                                        "Mail/Components/**",
+                                        "Mail/Helpers/**",
+                                        "Mail/Utils/**",
+                                        "Mail/Views/**",
+                                        "Mail/Proxy/Protocols/**"],
+                              resources: [
+                                  "MailShareExtension/Base.lproj/MainInterface.storyboard",
+                                  "MailShareExtension/ShareExtension.entitlements",
```

Comment:
```suggestion
```
This line copies the entitlement in the bundle but it's not needed we only need to make it available to Xcode which is already done by ln 134

Reply:
nice catch